nesc/whip6/lib/sys/bootseq/MainPub.nc

```diff
@@ -81,6 +81,9 @@ implementation {
         call SystemInitEntry.init();
         while(call Scheduler.runNextTask());
 
+        /* Trim VIMS to be set to either GPRAM or CACHE mode depending on flag */
+        trimVIMSMode();
+
```

Comment:
Question: May lines 79-82 also want to use GPRAM?

Reply:
As Mateusz suggested, VIMS should be configured earlier.